### PR TITLE
[Feature-3939][studio] FlinkSQL Studio supports real-time update task status

### LINKED ISSUE #3939: [Feature][studio] FlinkSQL Studio supports real-time update task status
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

_No response_

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by aiwenmo**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by aiwenmo** [comment]
image

**Comment by aiwenmo** [comment]
image

File: dinky-admin/src/main/java/org/dinky/ws/topic/TaskRunInstance.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    private Set<Integer> runningJobIds = CollUtil.newHashSet();
+
+    private TaskRunInstance() {}
+
+    @Override
+    public Map<String, Object> autoDataSend(Set<String> allParams) {
+        Set<Integer> currentMonitorTaskIds = FlinkJobThreadPool.getInstance().getCurrentMonitorTaskIds();
+        if (!runningJobIds.equals(currentMonitorTaskIds)) {
+            runningJobIds = currentMonitorTaskIds;
+            System.out.println("New IDS:" + runningJobIds.toString());
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/ws/topic/TaskRunInstance.java]: use slf4f??

Comment (aiwenmo) [reply dinky-admin/src/main/java/org/dinky/ws/topic/TaskRunInstance.java]: I will remove it.

File: dinky-web/src/pages/DataStudio/CenterTabContent/SqlTask/index.tsx [hunk 2]
```diff
@@ -757,7 +754,7 @@ export const SqlTask = memo((props: FlinkSqlProps & any) => {
               showDesc={showDesc}
               disabled={isLockTask}
               desc={l('button.offline')}
-              icon={<CloudDownloadOutlined style={{ color: 'red' }} />}
+              icon={<MergeCellsOutlined />}
```
Comment (Zzm0809) [inline dinky-web/src/pages/DataStudio/CenterTabContent/SqlTask/index.tsx]: rollback

File: dinky-web/src/pages/DataStudio/CenterTabContent/SqlTask/index.tsx [hunk 3]
```diff
@@ -729,7 +726,7 @@ export const SqlTask = memo((props: FlinkSqlProps & any) => {
             <Divider type={'vertical'} style={{ height: dividerHeight }} />
             <RunToolBarButton
               showDesc={showDesc}
-              disabled={isLockTask || JOB_LIFE_CYCLE.PUBLISH == currentState.step}
+              disabled={isLockTask}
```
Comment (Zzm0809) [inline dinky-web/src/pages/DataStudio/CenterTabContent/SqlTask/index.tsx]: rollback

**Review by zackyoungh — APPROVED** [review]
LGTM

**Pull request merged**